src/main/java/pl/touk/sputnik/processor/tslint/TSLintProcessorFactory.java

```diff
@@ -0,0 +1,18 @@
+package pl.touk.sputnik.processor.tslint;
+
+import pl.touk.sputnik.configuration.Configuration;
+import pl.touk.sputnik.configuration.GeneralOption;
+import pl.touk.sputnik.processor.ReviewProcessorFactory;
+
+public class TSLintProcessorFactory implements ReviewProcessorFactory<TSLintProcessor> {
```

Comment:
[Checkstyle] ERROR: Class 'TSLintProcessorFactory' must be declared as 'abstract'.